backends-common/elasticsearch/src/main/java/org/apache/james/backends/es/IndexCreationFactory.java

```diff
@@ -35,6 +35,9 @@
     private static final int DEFAULT_NB_SHARDS = 1;
     private static final int DEFAULT_NB_REPLICA = 0;
     public static final String CASE_INSENSITIVE = "case_insensitive";
+    public static final String KEEP_MAIL_AND_URL = "keep_mail_and_url";
+    public static final String SNOWBALL_KEEP_MAIL_AND_URL = "snowball_keep_mail_and_token";
```

Comment:
s/AND_URL/AND_TOKEN/ ?
It's very hard from these constant name to know what there are used for.

Reply:
I know but could not find other name, I am open to suggestion!

Reply:
I am open to suggestion